lib/rubocop/cop/style/arguments_forwarding.rb

```diff
@@ -186,46 +187,36 @@ def classification_and_forwards(def_node, send_node, referenced_lvars, forwardab
 
         def register_forward_args_offense(def_arguments_or_send, rest_arg_or_splat)
           add_offense(rest_arg_or_splat, message: ARGS_MSG) do |corrector|
-            unless parentheses?(def_arguments_or_send)
-              add_parentheses(def_arguments_or_send, corrector)
-            end
+            add_parens_if_missing(def_arguments_or_send, corrector)
 
             corrector.replace(rest_arg_or_splat, '*')
           end
         end
 
         def register_forward_kwargs_offense(add_parens, def_arguments_or_send, kwrest_arg_or_splat)
           add_offense(kwrest_arg_or_splat, message: KWARGS_MSG) do |corrector|
-            if add_parens && !parentheses?(def_arguments_or_send)
-              add_parentheses(def_arguments_or_send, corrector)
-            end
+            add_parens_if_missing(def_arguments_or_send, corrector) if add_parens
 
             corrector.replace(kwrest_arg_or_splat, '**')
           end
         end
 
-        def register_forward_all_offense_on_forwarding_method(forwarding_method)
```

Comment:
`register_forward_all_offense_on_forwarding_method` and `register_forward_all_offense_on_method_def` were essentially doing the same thing, and could be made to call the same method.